plugin_tests/parseSpec.js

```diff
@@ -0,0 +1,701 @@
+girderTest.importPlugin('jobs', 'worker', 'slicer_cli_web');
```

Comment:
Other than indentation, I assume nothing else is changed beyond the first 8 lines?